bliss/predict.py

```diff
@@ -11,7 +17,101 @@ def prepare_image(x, device):
     return x[:, :, :height, :width]
 
 
-def predict(cfg, image, background):
+def crop_plocs(cfg, h, plocs):
+    tiles_to_crop = cfg.encoder.tiles_to_crop
+    tile_slen = cfg.encoder.tile_slen
+    if not cfg.predict.ifSimulated:
+        crop = cfg.predict.crop
+        minb = crop[0] + tiles_to_crop * tile_slen
+        maxb = crop[1] - tiles_to_crop * tile_slen
+
+    else:
```

Comment:
Is there anyway to avoid this if-else statement, so that images are cropped in the same way regardless of where they are from? (I'm not sure there is, but it's good to avoid branches whenever possible: they're harder to test and harder to understand.)

Reply:
For example, if the simulated data were a little more like the real data, could we avoid having this branch?

Reply:
Yes, that seems good